scripts/process-releases.js

```diff
@@ -235,57 +235,74 @@ async function fixHtmlTags(dir) {
 
 async function fixBrokenLinks(dir) {
   // If you have warning of broken links, make sure the offending version is listed here
-  const versions = ['v1.14.x', 'v3.29.x', 'v4.0.x', 'v4.1.x', 'v4.18.x', 'latest']
-  const silent = true
-
-  // typo in the docs
-  replace({
-    regex: /Referece/g,
-    replacement: 'Reference',
-    paths: versions.map((version) => join(dir, `version-${version}/Guides`)).filter(fsOld.existsSync),
-    recursive: true,
-    silent,
-  })
-
-  // typo filename in the docs
-  replace({
-    regex: /Reference\/index/g,
-    replacement: 'Reference/Index',
-    paths: versions.map((version) => join(dir, `version-${version}/`)).filter(fsOld.existsSync),
-    recursive: true,
-    silent,
-  })
-
-  // double parenthesis in the docs
-  // ((../Guides/Getting-Started.md#your-first-plugin))
-  replace({
-    regex: /\((\(\.\.\/Guides\/Getting-Started\.md.*\))\)/g,
-    replacement: '$1',
-    paths: versions.map((version) => join(dir, `version-${version}/Reference`)).filter(fsOld.existsSync),
-    recursive: true,
-    silent,
-  })
+  const versions = [
+    'v1.14.x',
+    'v3.29.x',
+    'v4.0.x',
+    'v4.1.x',
+    'v4.2.x',
+    'v4.3.x',
+    'v4.4.x',
+    'v4.5.x',
+    'v4.6.x',
+    'v4.7.x',
+    'v4.8.x',
+    'v4.9.x',
+    'v4.10.x',
+    'v4.11.x',
+    'v4.12.x',
+    'v4.13.x',
+    'v4.14.x',
+    'v4.15.x',
+    'v4.16.x',
+    'v4.17.x',
+    'v4.18.x',
+    'v4.19.x',
+    'latest',
+  ]
```

Comment:
This doesn't really scale. We should not be changing this file every time we ship a new release of Fastify

Reply:
I agree. Fun fact, it was unnecessary: it now runs fixing on all directories in the versioned_docs folder.